### PR TITLE
createCardList

### PR DESCRIPTION
**Pull request by Leukipos**
also..
- adjust Style
- link the Button on landing Page

### CODE REVIEW

File: components/StyledButton.js [hunk 1]
```diff
@@ -1,30 +1,22 @@
...
   width: 8rem;
   height: 8rem;
   border-radius: 50%;
   background-color: #546391;
   padding: 0.8rem;
- 
   color: #EBE1CA;
   text-decoration: none;
   font-weight: bold;
   border: 0px;
```
Comment (NikRoe) [inline components/StyledButton.js:12]: no need for a unit here :) 

```suggestion
  border: 0;
```

File: components/StyledButton.js [hunk 2]
```diff
@@ -1,30 +1,22 @@
...
-  font-size: inherit;
+  font-size: 1.2rem;
   box-shadow: 0 0px 25px 2px #EBE1CA, 0 0px 5px 0px #ffffff, inset 0 0px 25px 2px #EBE1CA;
```
Comment (NikRoe) [inline components/StyledButton.js:14]: you can remove those units

```suggestion
  box-shadow: 0 0 25px 2px #EBE1CA, 0 0 5px 0 #ffffff, inset 0 0 25px 2px #EBE1CA;
```

File: pages/main.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+export default function CardList(){
+    return(
+        <ul>
+            <StyledCard>Modul Eins</StyledCard>
+            <StyledCard>Modul Zwei</StyledCard>
+            <StyledCard>Modul Drei</StyledCard>
+            <StyledCard>Modul Vier</StyledCard>
+            <StyledCard>Modul Fünf</StyledCard>
+            <StyledCard>Modul Sechs</StyledCard>
```
Comment (NikRoe) [inline pages/main.js:12]: you should consider making this part of the app dynamic at a later point and work with `.map()`

**Review by NikRoe — CHANGES_REQUESTED** [review]
Hi Alex,

this is a very promising start! 

I left only a few notes. Please take a look at those and implement them :)

**Comment by Leukipos** [comment]
Thank you for the help I solved the issues and i hope its fine now

**Review by NikRoe — APPROVED** [review]
LGTM 👍

**Pull request merged**